.github/workflows/build_and_tests.yml

```diff
@@ -1,41 +1,54 @@
 name: Build, test and deploy Ragger
 
 on:
+  workflow_dispatch:
   push:
     branches:
       - master
+      - main
       - develop
   pull_request:
-    branches:
-      - develop
-      - master
 
 jobs:
+  build_boilerplate_application:
+    name: Build boilerplate application using the reusable workflow
+    uses: LedgerHQ/ledger-app-workflows/.github/workflows/reusable_build.yml@fbe/more_options
+    with:
+      app_repository: LedgerHQ/app-boilerplate
+      app_branch_name: master
+      app_binaries_artifact: boilerplate_binaries
+
   build_install_test:
-    name: Build, install and test the library
+    name: Install and test the library
+    needs: build_boilerplate_application
     runs-on: ubuntu-latest
     env:
       PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION: python
```

Comment:
Can probably be removed now?

Reply:
Normally yes but I got burned too many times by the protobuf issues.
I'll remove it but if it comes back I'll just add the env again I won't debug

Reply:
Deal :)